clmm/modeling.py

```diff
@@ -542,7 +558,8 @@ def predict_reduced_tangential_shear(r_proj, mdelta, cdelta, z_cluster, z_source
                                     z_src_model)
         gamma_t = predict_tangential_shear(r_proj, mdelta, cdelta, z_cluster, z_source, cosmo,
                                            delta_mdef, halo_profile_model, z_src_model)
-        red_tangential_shear = gamma_t / (1 - kappa)
+        red_tangential_shear = np.nan_to_num(np.divide(gamma_t , (1 - kappa)))
```

Comment:
(1) Does the gamma_t/(1-kappa) no longer work here?
Can we explicitly add nan kwarg for readability:
`red_tangential_shear = np.nan_to_num(gamma_t / (1 - kappa), nan=0.0)`